scaleway/resource_server.go

```diff
@@ -41,6 +41,19 @@ func resourceScalewayServer() *schema.Resource {
 				Description:  "The instance type of the server",
 				ValidateFunc: validateServerType,
 			},
+			"boot_type": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ForceNew:     true,
+				Description:  "The boot_type of the server",
+				ValidateFunc: validateBootType,
```

Comment:
There’s a built-in validation function provided by Terraform Core that we can use here:

validation.StringInSlice([]string{
  “bootscript”,
  “local”,
}, true)

Where the true stands for case sensitive

Reply:
nice! didn't know about this.